web/components/app-portal-angular/projects/convoy-app/src/lib/convoy-app.component.ts

```diff
@@ -225,7 +222,7 @@ export class ConvoyAppComponent implements OnInit {
 
 		try {
 			const eventsResponse = await this.convyAppService.request({
-				url: this.getAPIURL(`/events?groupID=${this.groupId || ''}&sort=AESC&page=${this.eventsPage || 1}&perPage=20&startDate=${startDate}&endDate=${endDate}&appId=${this.appId || ''}`),
```

Comment:
@Oluwadaminiola Please why did you remove `perPage=20` here. Backend sending 20 by default?

Reply:
yes, I confirmed with backed that they had a default page count of 20 @horlah